ansible/roles/keystone/tasks/init_fernet.yml

```diff
@@ -17,31 +39,10 @@
   retries: 10
   delay: 5
   run_once: True
-  delegate_to: "{{ groups[keystone_services['keystone-fernet'].group][0] }}"
-  when: not use_preconfigured_fernet | bool
+  delegate_to: "{{ groups['keystone'][0] }}"
```

Comment:
Better to keep this the way it was before; assuming there is a `keystone` group seems sane but it makes a bit less flexible for people who are overriding things here (assuming we want to propose this upstream.)

Reply:
This was the delegation in the original, but your suggestion seems sensible.